Miscellaneous/scf-dictionary

```diff
@@ -0,0 +1,9 @@
+scf_guess = {
+    "EthaneA c-c": "8 molecule, 7 guess"
+    "PropaneA c-c-c": "8 molecule, 8 guess"
```

Comment:
For some reason I have a syntax error on line 3, at the colon.  

But yeah, these are the results. We didn't save much on cycles. If you curious; take ethane to be C'H3-C''H2-C'''H2-C''''H3 . I would take an ethane to be C' and C" (neighboring carbons), I would say the fragment as a whole has one unpaired electron at C''.

Reply:
you need a comma between pairs (at the end of line 2)

Reply:
(and at the end of line 3, 4, ... etc.)

Reply:
Ah. Thank you.